### PR TITLE
[WIP] CLI

### PR DESCRIPTION
**Pull request by kthai15**
To fire up the NodeJS app:
```
set `OPENAI_API_KEY=` in .env
npm install
node agent-cli.js --verbose
```

To kill the app, just type in `exit`.

### CODE REVIEW

**Comment by gravityrail** [comment]
I'm having trouble running this. Looks like it's missing `agents/toolkits/combined.js` and perhaps some other files.

**Comment by kthai15** [comment]
Yea, it's still pretty broken right now. I'll push up some changes in a bit.

**Comment by kthai15** [comment]
@gravityrail I think this is in a decent state now, if you want to give it a test.

**Comment by glendaviesnz** [comment]
working for me - just added not to the PR description to set `OPENAI_API_KEY` env var.

**Comment by glendaviesnz** [comment]
@kthai15 I made some changes to allow us to easily pass in a specific agent to the cli, so ` node ./agent-cli.js --verbose --agent=WPSiteSpec` now works. The site spec is the only one working properly as need to sort out the toolkits for the others.

But feel free to revert [this](https://github.com/Automattic/big-sky-agents/pull/15/commits/d2b1fc87b7c6edbd24dbdaea3f9d931d9ecc6498) if it messes up any plans you had for cli. 

We need to have a think about how to handle the toolkits, eg. in some cases we may not want to inherit the current agent toolkit when switching agents, but currently things break if an agent toolkit does not include `SimpleAgentToolkit`. There may be some instances when a flow might just end and not need to return to another agent.

Thanks for getting this started, it has really helped me to get my head around things better just being able to run the agent in isolation in cli like this.

File: src/agents/agent.js [hunk 1]
```diff
@@ -25,7 +24,7 @@ class Agent {
 	}
 
 	getAssistantId() {
-		return WAPUU_ASSISTANT_ID;
```
Comment (kthai15) [inline src/agents/agent.js]: @glendaviesnz was there a reason for changing the assistant ID?

Comment (kthai15) [reply src/agents/agent.js]: I'm going to change this back for now. I think this breaks the demo that uses the Assistants API, but the CLI still seems to be working.

**Pull request merged**